db/sql/030_create_billable.sql

```diff
@@ -74,13 +74,14 @@ event_ranges as (
 -- this results in a row per-instance
 resources as (
 	select
-		t.*,
-		generate_series(1, t.inst_count)
+		t.id || '-' || t.guid || '-' || generate_series(1, t.inst_count) AS billable_id,
```

Comment:
Do we really need to introduce a new column here? Is it not possible to keep the unique column as `id` rather than `billable_id`